bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -62,7 +62,7 @@
 
 		<label>UniFi WLAN</label>
 		<description>A UniFi Wireless LAN</description>
-
+		<semantic-equipment-tag>NetworkSwitch</semantic-equipment-tag>
```

Comment:
This is a logical Wi-Fi network, not a network switch.

Reply:
Ditto above.